docs/modules/servers/pages/local/self-hosting.notes

```diff
@@ -0,0 +1,90 @@
+The name of the project is "Jame Self Hosting Sandbox", and I see the term "self hosting" in a few places... So it sounds like it is important, but what is that intended to mean?
+
+I would like to understand better the intention behind the term.
+
+
+
+I would say "self hosting" means individuals having bought a domain name and want to install their own mail server for hosting their personal emails. Volumetry is small. They want control on their data.
+
+One generally tricky point about this kind of set up is getting third parties (like eg GMail) trusting your setup. It would be important to document a checklist regarding Mail Exchange trust like:
+
+ I'm not an open spam relay
+ MX record registered for my domain
+ Reverse DNS setted up for MX servers
+ SPF records setted up
+ DKIM signature setted up
+(which is slightly external to James as DNS setup is required)
+
+
+
+A "how to" regarding this is a MUST have in my opinion http://james.apache.org/howTo/index.html
+
+
+
+Just a thought based on the ideas you wrote here...
+
+But what if the mission of James were to "help people self-host their email" as an alternative to providers like gmail etc.
+
+Right now James is very obscure and is only for total tekkie geeks. If we modify the target, perhaps this could be the mission.
+
+The way of accomplishing this is:
+
+Provide in-depth "concept" articles
+Provide more "how tos" for important things
+Make James easier to install and operate
```

Comment:
Make James easier to install and operate => flavour foxussed documentation, simplified server is a step in that direction :-)

Reply:
Thanks. I added this comment to the notes.